packages/@lwc/ssr-compiler/src/compile-js/index.ts

```diff
@@ -23,6 +23,12 @@ import type { CompilationMode } from '@lwc/shared';
 
 const visitors: Visitors = {
     $: { scope: true },
+    ExportNamedDeclaration(path) {
+        replaceNamedLwcExport(path);
```

Comment:
Is `replaceNamedLwcExport` used anywhere else? Can we just inline it here?

Reply:
It is not, but same goes for [replaceLwcImport](https://github.com/salesforce/lwc/pull/5034/files#diff-7c25a9ea23e88b7f09bbc4b7ac5231ccc37f5a4dccd9a82c8d62a8fa180f532dL31) so I was just keeping it consistent. Also feel like this file could get a little busy if we added everything inline?